CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.0] - 2021-02-01
```

Comment:
I don't think should be a "major" bump. The PR does not contain anything but the bump. It means the API is not affected whatsoever, so from the consumer of PKI's there is no changes at all.

It should be a "minor", unless I'm missing something.

Reply:
PKI is only using the `sponge::hash` of poseidon and I believe that is not affected by the changes landed in the latest Poseidon, right?

Reply:
Yep, version changed to minor